test/evaluator/Evaluator-throw-on-missing-prop-mode.js

```diff
@@ -0,0 +1,209 @@
+/*
+ * Jexl
+ * Copyright (c) 2015 TechnologyAdvice
```

Comment:
I don't know that this copyright is correct because it is new code?

Reply:
Updated.